build/mark-new-version.sh

```diff
@@ -92,6 +92,10 @@ fi
 
 echo "+++ Running ./versionize-docs"
 ${KUBE_ROOT}/build/versionize-docs.sh ${NEW_VERSION}
+
+echo "+++ Updating swagger"
+${KUBE_ROOT}/hack/update-generated-swagger-docs.sh
```

Comment:
Sorry I missed that. 

I think we need to run `update-swagger-spec.sh` after `update-generated-swagger-docs.sh`.